kolibri/auth/filters.py

```diff
@@ -0,0 +1,185 @@
+from django.db import models
+from django.db.models.query import F
+from six import string_types
+
+from .constants import collection_kinds
+
+
+class HierarchyRelationsFilter(object):
```

Comment:
I agree it would be good to have selective tests run with postgres -- I'm just not familiar enough with CircleCI to know how hard this is to set up (obviously we don't want every user to need to have postgres installed locally in order to run tests, but I imagine there's some mechanism for spinning up a postgres database within CircleCI, and then we can have those tests run selectively there). If you think that's easy to set up, let's do it (I'd need direction). Otherwise, I'd say let's cross that bridge when we get there.